2.6_chiseltest.ipynb

```diff
@@ -503,6 +503,36 @@
     "}\n"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "// test for backpressure\n",
+    "test(new DecoupledGcd(16)) { dut =>\n",
+    "  dut.input.initSource().setSourceClock(dut.clock)\n",
+    "  dut.output.initSink().setSinkClock(dut.clock)\n",
```

Comment:
What I want to do is to use slower clock for the sink so that there is greater chance that a `fork` + `join` approach runs into a backpressure scenario but I don't know how to write a ClockDivider here?